app/controllers/application_controller.rb

```diff
@@ -49,6 +49,10 @@ def current_user  # Override Devise helper method (controller instance method)
     super || Visitor.new
   end
 
+  def admins
```

Comment:
@weedySeaDragon This method was for fetching all the admins. I've moved this to `User` so I can do `User.admins` in the controller. Please correct me if I have misunderstood. Thanks.